CHANGELOG.md

```diff
@@ -4,27 +4,23 @@ All notable changes to this crate are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [2.2.0] - 2021-07-02
+
+### Added
 
-## [2.2.0-rc.3] - 2021-06-25
 - Add support for custom where bounds `codec(mel_bound(T: MaxEncodedLen))` when deriving the traits. PR #279
+- `MaxEncodedLen` trait for items that have a statically known maximum encoded size. ([#268](https://github.com/paritytech/parity-scale-codec/pull/268))
+- `#[codec(crate = "<path>")]` top-level attribute to be used with the new `MaxEncodedLen`
+trait, which allows to specify a different path to the crate that contains the `MaxEncodedLen` trait.
+Useful when using generating a type through a macro and this type should implement `MaxEncodedLen` and the final crate doesn't have `parity-scale-codec` as dependency.
 
-## [2.2.0-rc.2] - 2021-06-24
+### Changed
 
 - Updated syntax of `#[codec(crate = <path>)]` attribute macro: no longer expects the crate path to
```

Comment:
ah but this comment is about changing the syntax newly introduced. We should merge with the above added comment